src/middleware/language.js

```diff
@@ -39,7 +39,7 @@ function findExactLanguageMatch(languages, parsedLanguages) {
       }
       return matchingLanguage;
     });
-  return matchingLanguage ? languageObjectToString(matchingLanguage) : null;
+  return matchingLanguage || null;
```

Comment:
Out of the scope of that PR `_.find` is wrongly used as a `_.forEach` here. Nothing is done with the value he return. We should affect the value returned with the function instead.

Reply:
agreed, language mdw will be updated later on.